cmd/goal/application.go

```diff
@@ -190,7 +190,6 @@ func init() {
 
 	panicIfErr(methodAppCmd.MarkFlagRequired("method"))
 	panicIfErr(methodAppCmd.MarkFlagRequired("from"))
-	panicIfErr(appCmd.PersistentFlags().MarkHidden("app-arg"))
```

Comment:
this should not be in this PR since it is already in the feature branch

Reply:
@algorandskiy I'm not sure I'm following - Are you suggesting the diff should not appear?

From my side, it feels like the PR is accurate.  The highlighted line exists in `feature/avm-box` (https://github.com/algorand/go-algorand/blob/45b7d6081597562e6c77e2c2113202b547b9a159/cmd/goal/application.go#L192-L193), and is removed from `master`.  So, the PR shows a removal.

Reply:
right, I confused this with another PR